### DISCUSSION TITLE
[With HOW-TO] Please tell me your favourite MUSIC ALBUM💿🎤

### DISCUSSION THREAD

**Discussion by takuma1229** [NAIST-SE/AttractingContributors#1043] [2024 Attraction Contributions]
I would like you to add your favorite MUSIC ALBUM to the list in the README and make Pull Request 🙏 

## Link
- https://github.com/takuma1229/AttractingContributors-MusicAlbum/blob/main/README.md

## How to make Pull Request to my repository
1. On your editor, move to your repository which you forked from NAIST-SE
2. `git remote add music-album https://github.com/takuma1229/AttractingContributors-MusicAlbum.git`
3. `git fetch music-album`
4. `git checkout -b music-album-YOURNAME music-album/main`
5. ADD YOUR FAVOURITE MUSIC ALBUM.
6.  `git add README.md`
7. `git commit -m "add album"`
8. `git push origin music-album-YOURNAME`
9. Move to [Here](https://github.com/takuma1229/AttractingContributors-MusicAlbum/blob/main/README.md) and click "Compare and pull request" button.
10. Push "Create Pull Request" button
11. Finish!

**Comment by KiyotadaMori** [comment]
Hi, takuma1229. 
I sent PR about the topic. Please add my favorite album 「Parsley, Sage, Rosemary and Thyme」. 
Regards,

**Comment by gill876** [comment]
My favorite album is Born This Way by Lady Gaga!
Thanks for the ingenious instruction format!

Please contribute to my project!
https://github.com/gill876/AttractingContributors-Hiking

**Comment by oyewaleoyebode** [comment]
Made of Gold

**Reply by takuma1229** [reply]
Thanks! It's super cool if you send pull request to my repository:)
